content/en/continuous_delivery/deployments/argocd.md

```diff
@@ -196,11 +200,12 @@ To associate source code Git information with the Argo CD deployments, run the `
     git push
 ```
 
-Even if the same repository serves as the source code repository and configuration repository, running this command is still required since it also gathers the information required to perform the correlation between the pipeline and the deployment.
+**Note**: Even in case a single repository is used to store both the source code and the Kubernetes manifest, running this command is still required to correctly associate deployments and CI pipelines.
+
 
 ### Validation
 
-After successfully running the command, deployments contain Git metadata from the application repository instead of the configuration repository, and also display the associated pipeline. See the [Visualize correlated pipelines and deployments][18] section on the CD Visibility page for more details.
+If the command has been correctly run, deployments contain Git metadata from the application repository instead of the configuration repository. Also, the deployment executions view will contain a new **Pipeline** tab representing the related CI pipeline trace.
```

Comment:
```suggestion
If the command has been correctly run, deployments contain Git metadata from the application repository instead of the configuration repository. Also, the deployment executions view now contains a new **Pipeline** tab representing the related CI pipeline trace.
```